types/newrelic/config.d.ts

```diff
@@ -0,0 +1,836 @@
+export {};
+
+declare module "./config" {
```

Comment:
Declaring a module like this is definitely not desirable. You're basically saying "there is a package named `./config`" which is certainly not true.